DEVELOPER_GUIDE.md

```diff
@@ -39,7 +49,15 @@ node.max_local_storage_nodes: 3
 ######## End OpenSearch Security Demo Configuration ########
 ```
 
-Next, we need to check out OpenSearch Dashboards from the branch matching our version, in this example that would be the [1.x branch](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/1.x). Follow the [developer guide](https://github.com/opensearch-project/OpenSearch-Dashboards/blob/1.x/DEVELOPER_GUIDE.md) and replace the version of  `opensearch-dashboards.yml` there with this:
+
+**Please Note** : This project runs on node `10.24.1` (refer to the `.nvmrc` or `.node-version` file in the base directory for correct version) and so when installing node please ensure that you install this version. You can do so by running 
+```script 
+nvm install 10.24.1
```

Comment:
```suggestion
nvm use --install
```
That way we don't have to specify the version explicitly, since it is picked up from `.nvmrc`

Reply:
What's the deal with this comment?

Reply:
Just an idea for an improvement on the docs. E.g. if we move towards 2.0 the node version changed and that would mean we always have to update the docs when we change `.nvmrc`. If we simply use the `nvm use --install` command we then don't have to upgrade the docs all the time.